sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -290,7 +290,7 @@ interval 4 weeks 2 days
 -- !query 22
 select 30 day day
 -- !query 22 schema
-struct<>
+
```

Comment:
Sorry, but let's keep the original form. The first contribution seems to be on the edge.

Reply:
Thanks @dongjoon-hyun for the review, let me try to convince you that these changes make sense, but if you still disagree just let me know and I will drop them.

The only cases where I replaced the expected schema from `struct<>` to nothing are those where some error occurs and an exception is thrown. In those cases there is no data returned, so there is no schema at all, not even an empty struct.
```
-- !query 22
select 30 day day
-- !query 22 schema

-- !query 22 output
org.apache.spark.sql.catalyst.parser.ParseException
```
IMHO `struct<>` makes sense where a statement was successful but data returned is empty and there are no columns in it. In those cases I left the expected output intact.
```
-- !query 0
CREATE OR REPLACE TEMPORARY VIEW view1 AS SELECT 2 AS i1
-- !query 0 schema
struct<>
-- !query 0 output
```
Empty expected schema is also useful to easily recognize a statement that ended up in an error (otherwise we probably need to check the output for containing `exception` which seems less elegant).
I utilized empty expected schema to fix a sorting issue in `ThriftServerQueryTestSuite`: https://github.com/apache/spark/pull/26028/files#diff-b3ea3021602a88056e52bf83d8782de8R147, but here might be other cases in the future where this could help.

Reply:
I fully understood that, but it's not worth of this huge change.

Reply:
All right, I've dropped that part of changes.